src/ripple/shamap/SHAMap.h

```diff
@@ -82,7 +82,7 @@ class SHAMap
     Family&                         f_;
     beast::Journal                  journal_;
     std::uint32_t                   seq_;
-    std::uint32_t                   ledgerSeq_; // sequence number of ledger this is part of
+    std::uint32_t mutable           ledgerSeq_; // sequence number of ledger this is part of
```

Comment:
We should make the change you suggested. It was marked `mutable` for legacy reasons. Reasons which are no longer valid now that the code has changed.